src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,4 @@
+$blue-hover: #00acdc;
+$grey-star-color: #f3f3f3;
+$standart-inline-pad: 16px;
+$standart-black-color: #060b35;
```

Comment:
The variable '$standart-black-color' also has a typo. Consider renaming it to '$standard-black-color' for consistency and correct spelling.